tests/Unit/Services/Geocoder/GeocoderTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tests\Unit;
```

Comment:
This isn't an unit test, could you move it to an integration or acceptance test. It is just a naming thing, but I like my units being isolated. If  Positionstack changes it's api, I would expect unit tests to succeed and integration to fail.